### PR TITLE
fix vimhelplint violations

### PR DESCRIPTION
**Pull request by davidosomething**
@blueyed does vimhelplint have some inline ignore directives? That's the only reason I changed the logo

#600

### CODE REVIEW

**Comment by blueyed** [comment]
> does vimhelplint have some inline ignore directives? That's the only reason I changed the logo

I don't think so, but /cc @machakann.

It would probably look ugly inline though anyway.
We could just filter it out with `grep -v` though.

The new logo is a bit ugly IMHO - what about using the previous one, but with one line less, so it gets more compressed?

What program have you used to generate it?

```
  \  |                                 |          
   \ |   _ \   _ \   __ `__ \    _` |  |  /   _ \ 
 |\  |   __/  (   |  |   |   |  (   |    <    __/ 
_| \_| \___| \___/  _|  _|  _| \__,_| _|\_\ \___|
```

(via `toilet -w 80 -f fonts/shadow.flf Neomake`)

Or
```
    _   __                                __       
   / | / /___   ____   ____ ___   ____ _ / /__ ___ 
  /  |/ // _ \ / __ \ / __ `__ \ / __ `// //_// _ \
 / /|  //  __// /_/ // / / / / // /_/ // ,<  /  __/
/_/ |_/ \___/ \____//_/ /_/ /_/ \__,_//_/|_| \___/ 
```
(via `figlet -k Neomake`)

File: doc/neomake.txt [hunk 1]
```diff
@@ -109,10 +111,10 @@ usable on an individual file, you should also include the filtype in the name: >
     " working directory (runs "lint --option x"):
     :Neomake! c_lint
 <
-                                   *neomake-{{makepath}}* *neomake-!!makepath!!*
+                                 *neomake-{{makepath}}* *neomake-!!makepath!!*
```
Comment (blueyed) [inline doc/neomake.txt]: Please split it in two lines, so it lines up properly in any conceal level.

File: doc/neomake.txt [hunk 2]
```diff
@@ -29,51 +29,52 @@ run this plugin in both vim and neovim. This plugin is heavily inspired by
 fantastic vim plugins such as syntastic and dispatch.
 
 ==============================================================================
-2. Commands                                                   *neomake-commands*
+2. Commands                                                 *neomake-commands*
 
-                                                       *:Neomake* *:NeomakeFile*
+                                                     *:Neomake* *:NeomakeFile*
```
Comment (blueyed) [inline doc/neomake.txt]: Please split it in two lines, so it lines up properly in any conceal level.

**Comment by davidosomething** [comment]
i used this (click "test all" button to see all variants)
http://patorjk.com/software/taag/#p=display&f=Graffiti&t=Type%20Something%20

maybe @benekastah should pick xp

File: doc/neomake.txt [hunk 3]
```diff
@@ -29,51 +29,52 @@ run this plugin in both vim and neovim. This plugin is heavily inspired by
...
 :Neomake [makers]       Run a make command with the current file as input. If
                         no makers are specified, the default makers for the
-                        current |filetype| are used. See |neomake-configuration|
-                        for more on makers.
+                        current |filetype| are used. See
+                        |neomake-configuration| for more on makers.
 
-                                                     *:Neomake!* *:NeomakeProject*
+                                                 *:Neomake!* *:NeomakeProject*
```
Comment (blueyed) [inline doc/neomake.txt]: Please split it in two lines, so it lines up properly in any conceal level.

**Comment by blueyed** [comment]
Please use `Fixes https://github.com/neomake/neomake/issues/600` in the commit message to close the issue automatically.

**Comment by blueyed** [comment]
And please remove `vimhelplint` from the allowed failures (https://github.com/neomake/neomake/blob/7b64d071c910b6c1f06ffe8c3d1b9a9f5a26d097/.travis.yml#L36).

**Comment by machakann** [comment]
Hi, I'm an author of vimhelplint.

Unfortunately, there is no inline directives for vimhelplint currently because there is no "comment" in vimhelp. Some alternative ways to ignore error may be implemented for future.

**Comment by blueyed** [comment]
@machakann 
Thanks for your feedback!

I've created https://github.com/neomake/neomake/issues/690 for the bike-shedding part of this PR.. :)

File: doc/neomake.txt [hunk 4]
```diff
@@ -197,18 +200,21 @@ property: >
 
 Global Options                                               *neomake-options*
 
-*g:neomake_<name>_maker* *g:neomake_<ft>_<name>_maker*
+*g:neomake_<name>_maker*
+*g:neomake_<ft>_<name>_maker*
```
Comment (blueyed) [inline doc/neomake.txt]: Those would be OK, since not right-aligned.  But makes sense anyway.

**Comment by blueyed** [comment]
There seem to be two issues left:

> doc/neomake.txt:25:29:Error:4:A link "job-control" does not have a corresponding tag.
doc/neomake.txt:260:2:Error:4:A link "job-control" does not have a corresponding tag.

That's because it gets run with an older Vim (the one on Travis).  I will fix that, by running it through Docker.

**Comment by blueyed** [comment]
Please rebase it on master, then it should be green again.

**Comment by blueyed** [comment]
Rebased in https://github.com/neomake/neomake/pull/692.
Please review.
You should be able to push to that branch also.

**Pull request closed without merging**